.github/pull_request_template

```diff
@@ -0,0 +1,37 @@
+# Pull Request Guidelines
+
+## Please consider splitting up big PRs (more than 10-15 files) into several small ones for easier reviews
+
+---
+
+# Pull Request Template
+
```

Comment:
We could consider the following:
Title of the pull request (a short title that describes the changes).

Reply:
I will add that. Thank you for the input

Reply:
I believe we don't need it because there's the actual PR title.
which makes line 9 redundant.